src/main.js

```diff
@@ -52,17 +54,22 @@ let isNeptuneIAMAuth = false;
 let createUpdatePipeline = false;
 let createUpdatePipelineName = '';
 let createUpdatePipelineEndpoint = '';
+let createUpdatePipelineEndpointRO = '';
```

Comment:
Seems this new option was added to allow the user to specify a Neptune read only endpoint however I only see logic to eventually assign the value to NEPTUNE_HOST_RO and NEPTUNE_PORT_RO which is never referenced. We should ask Andrea N if he wants us to remove this or finish the implementation.

Reply:
Removed 🙆‍♀️